packages/babel-parser/test/fixtures/core/regression/octal-float-fail/options.json

```diff
@@ -1,3 +1,3 @@
 {
   "throws": "Unexpected token, expected \";\" (1:2)"
-}
\ No newline at end of file
+}
```

Comment:
Can you please remove all the unnecessary whitespace changes from this PR?

Reply:
Ah sorry, this got autogenerated during the test script.